Kernel/Syscalls/jail.cpp

```diff
@@ -14,6 +14,18 @@ namespace Kernel {
 
 constexpr size_t jail_name_max_size = 50;
 
+static ErrorOr<NonnullRefPtr<Jail>> create_jail_with_default_isolation_options(NonnullOwnPtr<KString> jail_name)
+{
+    return TRY(Jail::create(move(jail_name), Jail::EnforcePIDIsolation::Yes));
+}
+
+static ErrorOr<NonnullRefPtr<Jail>> create_jail_with_custom_isolation_options(NonnullOwnPtr<KString> jail_name, unsigned flags)
```

Comment:
```cpp
static ErrorOr<NonnullRefPtr<Jail>> create_jail_with_custom_isolation_options(NonnullOwnPtr<KString> jail_name, u32 flags)
```
This reflects the type in use for `JailIsolationFlags`. Should alleviate the need for all the `static_cast`s I think?